aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -188,6 +192,16 @@ module aptos_framework::stake {
         total_joining_power: u128,
     }
 
+    struct PendingTransactionFee has key, store {
+        pending_fee_by_validator: OrderedMap<address, Aggregator<u64>>,
```

Comment:
this is more than 4kb in a single slot? we only update one per block for now? or using validator index can be smaller?

does it need to be aggregator? it's only updated at block epilogue not every txn?

Reply:
aggregator is for state sync, so they don't get conflict every block.

Reply:
I think block metadata conflicts anyway

Reply:
updated to BigOrderedMap to workaround an exist limitation.